### PR TITLE
prep for map dumping

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Review by ethteck — APPROVED** [review]

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 8 functions (+0.07%) on `us`.

✅ This PR fixes 13 warnings!

File: src/world/script_api/7E0E80.c [hunk 1]
```diff
@@ -252,6 +244,11 @@ ApiStatus ShowShopPurchaseDialog(Evt* script, s32 isInitialCall) {
     s32 bpCost;
     s32 args;
 
+    static MessagePrintState* D_80286528;
```
Comment (ethteck) [inline src/world/script_api/7E0E80.c:247]: these should go to the top of the file

File: src/world/script_api/7E0E80.c [hunk 2]
```diff
@@ -470,161 +467,187 @@ ApiStatus ShowShopOwnerDialog(Evt* script, s32 isInitialCall) {
     GameStatus* gameStatus = gGameStatusPtr;
     PlayerData* playerData = &gPlayerData;
     Shop* shop = gameStatus->mapShop;
-    ItemData* item;
-    s32 temp_v1_2;
+
+    static MessagePrintState* D_80286538;
```
Comment (ethteck) [inline src/world/script_api/7E0E80.c:471]: this too

File: src/world/script_api/7E2AA0.c [hunk 3]
```diff
@@ -1,31 +1,45 @@
...
+	/* 0x3C */ s32 modelID;
+	/* 0x40 */ s32 bgColor[3];
+	/* 0x4C */ s32 scriptIDs[4];
+	/* 0x5C */ s32* npcList;
+	/* 0x60 */ s32 colliderID;
+} AdvancedDoor; // size = 0x64
 
 ApiStatus func_80281C20(Evt* script, s32 isInitialCall) {
-    Bytecode* thisPos = script->ptrReadPos;
+    Bytecode* args = script->ptrReadPos;
```
Comment (ethteck) [inline src/world/script_api/7E2AA0.c:40]: lol wow, I haven't named one of those `thisPos` in AGES. this must be a dinosaur decomp

**Review by ethteck — DISMISSED** [review]

File: src/world/script_api/7E2AA0.c (hunk 3 above)
Comment (HailSanta) [reply src/world/script_api/7E2AA0.c:40]: Yeah, world and evt api were probably some of the first ones to be done.

File: include/script_api/macros.h [hunk 4]
```diff
@@ -195,6 +195,8 @@
 /// Blocks for the given number of frames.
 #define EVT_WAIT_FRAMES(NUM_FRAMES)             EVT_CMD(EVT_OP_WAIT_FRAMES, NUM_FRAMES),
 
+#define EVT_WAIT(NUM_FRAMES)             EVT_CMD(EVT_OP_WAIT_FRAMES, NUM_FRAMES),
```
Comment (nanaian) [inline include/script_api/macros.h:198]: Would be better to remove WAIT_FRAMES entirely than just alias I think

**Review by nanaian — DISMISSED** [review]

File: include/script_api/macros.h (hunk 4 above)
Comment (HailSanta) [reply include/script_api/macros.h:198]: Will do in a second pass with mass-renaming. Don't want to touch all those files at the moment.

File: include/evt.h [hunk 5]
```diff
@@ -120,6 +120,33 @@ enum {
...
+#define VAR9 EVT_VAR(9)
+#define VARA EVT_VAR(10)
+#define VARB EVT_VAR(11)
+#define VARC EVT_VAR(12)
+#define VARD EVT_VAR(13)
+#define VARE EVT_VAR(14)
+#define VARF EVT_VAR(15)
+
 // deprecated, use enum GSW/GSWF
 #define EVT_STORY_PROGRESS EVT_SAVE_VAR(0)
```
[the remark this answers is not in the document]
Comment (HailSanta) [reply include/evt.h:151]: Yeah, I'll do that when we rename all the GFSW etc macros and rename all the EVT_SAVE_FLAG(X) etc across the codebase. Will do in a separate PR.

**Review by ethteck — APPROVED** [review]

**Pull request merged**